crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -492,6 +494,7 @@ async fn deploy_kakarot(
         *class_hash.get("contract_account").unwrap(),
         *class_hash.get("externally_owned_account").unwrap(),
         *class_hash.get("proxy").unwrap(),
+        *DEPLOY_FEE,
```

Comment:
i kinda still don't get the function of this constant? like why did we introduce it in the first place

Reply:
I'm not too sure, I think @bajpai244 has more context